### PR TITLE
Add org.winehq.Wine.DLLs.dxvk

### PR DESCRIPTION
**Pull request by gasinvein**
A DXVK extension for the Wine flatpak app.

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by gasinvein** [comment]
bot, build org.winehq.Wine.DLLs.dxvk

File: org.winehq.Wine.DLLs.dxvk.yml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+sdk: org.freedesktop.Sdk//21.08
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.mingw-w64
+build-options:
+  prefix: /app/dlls/dxvk
+  prepend-path: /app/dlls/dxvk/bin
+  append-path: /usr/lib/sdk/mingw-w64/bin
+  #FIXME -fstack-clash-protection causes ICE
+  # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=90458
+  cflags: -O2 -g -pipe
```
Comment (Erick555) [inline org.winehq.Wine.DLLs.dxvk.yml:14]: Would passing  `-fno-stack-clash-protection` be sufficient?

Comment (gasinvein) [reply org.winehq.Wine.DLLs.dxvk.yml]: Looks like it, thanks for the hint.

**Comment by gasinvein** [comment]
bot, build org.winehq.Wine.DLLs.dxvk

File: org.winehq.Wine.DLLs.dxvk.yml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  - org.freedesktop.Sdk.Extension.mingw-w64
+build-options:
+  prefix: /app/dlls/dxvk
+  prepend-path: /app/dlls/dxvk/bin
+  append-path: /usr/lib/sdk/mingw-w64/bin
+  #FIXME -fstack-clash-protection causes ICE
+  # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=90458
+  cflags: -fno-stack-clash-protection
+  cxxflags: -fno-stack-clash-protection
+  ldflags: ''
```
Comment (Erick555) [inline org.winehq.Wine.DLLs.dxvk.yml:16]: What's the reason behind clearing ldflags? Does it fail with default set (`-Wl,-z,relro,-z,now -Wl,--as-needed`)?

Comment (gasinvein) [reply org.winehq.Wine.DLLs.dxvk.yml:16]: It does, mingw linker doesn't understand `-z`.

**Comment by gasinvein** [comment]
Oh, this should've go to the usual git `master` branch, since the extension contains only PE files and doesn't depend on any ABI in the runtime. @barthalion can you please change the default branch to `master`?

**Pull request closed without merging**